pkg/operation/seed/components.go

```diff
@@ -293,49 +296,56 @@ func defaultDependencyWatchdogs(
 	}
 
 	var (
-		dwdEndpointValues = dependencywatchdog.BootstrapperValues{Role: dependencywatchdog.RoleEndpoint, Image: image.String()}
-		dwdProbeValues    = dependencywatchdog.BootstrapperValues{Role: dependencywatchdog.RoleProbe, Image: image.String()}
+		defaultWatchDuration = pointer.Duration(5 * time.Minute)
```

Comment:
duration should be a const and then in the place you are using it create a pointer out of it by using `pointer.Duration`